### PR TITLE
Upgrading pulumi-kubernetes and adding tests, fixes #40

### LINKED ISSUE #40: Support `pulumi-kubernetes` >= 4.0.0
## Hello!

- Vote on this issue by adding a 👍 reaction
- If you want to implement this feature, comment to let us know (we'll work with you on design, scheduling, etc.)

## Issue details

This repo only supports `pulumi-kubernetes` < 4.0.0, which is now very old, `pulumi-kubernetes` is on `4.11.0` already. I cannot use this package without support for the latest `pulumi-kubernetes`, since I do not want to downgrade my `pulumi-kubernetes` package.

### Affected area/feature

`pulumi-kubernetes` integration

### PR DESCRIPTION
**Pull request by rshade**
This PR Updates the following packages:

- pulumi/pulumi - 3.143.0
- pulumi/pulumi-kubernetes - 4.19.0
- pulumi/pulumi-go-helmbase - 0.1.0

It also moves the schema.json to the same location as cert-manager, and copies over the tests structure. It currently only tests typescript. This gets it to a much more stable position to fix the other issues.

### CODE REVIEW

File: examples/examples_py_test.go [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+	basePython := base.With(integration.ProgramTestOptions{
+		Dependencies: []string{
+			filepath.Join("..", "sdk", "python", "bin"),
+		},
+	})
+
+	return basePython
+}
+
+// func TestPyExamples(t *testing.T) {
```
Comment (mjeffryes) [inline examples/examples_py_test.go:24]: I saw you updated the python example too. Should we uncomment this to run it?

**Review by mjeffryes — COMMENTED** [review]
Thanks for doing this! It will be great to have this under ci-mgmt going forward. hopefully less likely to fall behind again!

File: examples/examples_py_test.go (hunk 1 above)
Comment (rshade) [reply examples/examples_py_test.go:24]: I wasn't sure how the python examples get the right version of the package. I couldn't get it to pull the right version locally without a `-e ../sdk/python` and I didn't want to leave that in the example. If it should `just work` then I am good to test it as well.

Comment (mjeffryes) [reply examples/examples_py_test.go:24]: I think maybe line 17 needs to be `filepath.Join("..", "sdk", "python", "pulumi_kubernetes_ingress_nginx")`? 

This is always a little tricky to get right! I'm fine with leaving it disabled if you can't work it out.

Comment (rshade) [reply examples/examples_py_test.go:24]: Cool, thank you. I will check it out, and let you know early next week.

Comment (rshade) [reply examples/examples_py_test.go:24]: I tried a few different things. I am going to let this go in, release, then re-enable the python test.

**Review by blampe — APPROVED** [review]

**Pull request merged**